packages/libz/libz.spec

```diff
@@ -1,5 +1,5 @@
 Name: %{_cross_os}libz
-Version: 1.2.13
+Version: 1.3
```

Comment:
I don't see anything of note on why they went with 1.3 vs 1.2.14 but I'd like to keep an eye on things using this incase there is something new in here. They last did a bump to 1.2.x March 9, 2003 so this is worth digging into for a minute.

Reply:
I was able to successfully  run a container in k8s cluster.